simulator/simulator.cpp

```diff
@@ -1,81 +1,148 @@
 #include "simulator/simulator.h"
+#include "absl/strings/str_join.h"
+#include "absl/strings/str_split.h"
+#include "absl/strings/numbers.h"
 #include "glog/logging.h"
 
 namespace et_simulator {
-void Simulator::read_trace_file(std::ifstream & in) {
-  int64_t record_count = 0;
-  char kind;
-  std::string line;
+bool Simulator::parse_object_allocation(std::vector<std::string> trace) {
+  if(trace.size() != 5) {
```

Comment:
Missing a space after if (also in other parse_ methods).

Reply:
Fixed the missing space in the if statements